Dangerfile

```diff
@@ -0,0 +1,22 @@
+message(“Thank you for submitting a pull request to SwifterSwift. The team will review your submission as soon as possible.“)
```

Comment:
YOU'RE BETRAYING DANGER-SWIFT WITH THIS PEACE OF RUBY? 😭

Reply:
🤣 You can see where my heart was in the previous commit. I can't even write Ruby. 
danger-swift is still blocked by Marathon but it's close!

Reply:
Hah, yeah, if anyone wants to try help on that it's a real fun one  -https://github.com/JohnSundell/Marathon/pull/127